ci/scripts/git_skip_ci_globs.py

```diff
@@ -35,6 +35,8 @@
     # microTVM
     "apps/microtvm/poetry.lock",
     "apps/microtvm/pyproject.toml",
+    "tests/lint/*",
+    "tests/scripts/task_lint.sh",
```

Comment:
Why you added these files to the `globs`?

Reply:
These are used to determine in Jenkins whether to run the full suite of tests or not, so if all the files in a PR match one of these globs then unit tests won't be run (lint is always run though). These files only affect lint so they don't need to run tests

Reply:
Got it. Thank you! I just thought that lint also won't be run. And I was confused about that.